src/modules/battery.cpp

```diff
@@ -313,10 +384,12 @@ const std::string waybar::modules::Battery::formatTimeRemaining(float hoursRemai
 }
 
 auto waybar::modules::Battery::update() -> void {
+#if __Linux__
```

Comment:
Witthout `defined` may trigger `-Wundef` on FreeBSD and should be lowercase:
```suggestion
#if defined(__linux__)
```